.github/workflows/pr-push.yml

```diff
@@ -18,3 +18,4 @@ jobs:
           W3C_WG_DECISION_URL: https://lists.w3.org/Archives/Public/public-webappsec/2015Mar/0170.html
           W3C_BUILD_OVERRIDE: |
             shortname: FileAPI
+            specStatus: WD
```

Comment:
actually looks like perhaps this should just be status rather than specStatus?

Reply:
> actually looks like perhaps this should just be status rather than specStatus?

oops — thanks, yes, changed now